### PR TITLE
Migrate to beta-v2

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

File: tools/dataProtector/dataProtectorCore/protectData.md [hunk 1]
```diff
@@ -153,14 +153,14 @@ const protectedData = await dataProtectorCore.protectData({
...
-'DEPLOY_PROTECTED_DATA'
-'PUSH_SECRET_TO_SMS'
+```ts
+'EXTRACT_DATA_SCHEMA';
+'CREATE_ZIP_FILE';
+'CREATE_ENCRYPTION_KEY';
+'ENCRYPT_FILE';
+'UPLOAD_ENCRYPTED_FILE';
+'DEPLOY_PROTECTED_DATA';
+'PUSH_SECRET_TO_SMS';
```
Comment (cedric25) [inline tools/dataProtector/dataProtectorCore/protectData.md:163]: Hum that's not really TypeScript here.
Why not keep it the way it was?

File: tools/dataProtector/dataProtectorCore/grantAccess.md [hunk 2]
```diff
@@ -218,43 +218,14 @@ const grantedAccess = await dataProtectorCore.grantAccess({
...
-
-4️⃣
-
-```json
-{
-  "title": "PUBLISH_DATASET_ORDER",
-  "isDone": true
-}
-```
+Once with `isDone:false`, and then with `isDone:true`
```
Comment (cedric25) [inline tools/dataProtector/dataProtectorCore/grantAccess.md:228]: Missing spaces:
```
Once with `isDone: false`, and then with `isDone: true`
```

I'll push that :)

File: tools/dataProtector/dataProtectorCore/processProtectedData.md [hunk 3]
```diff
@@ -210,6 +210,45 @@ const taskid = await dataProtectorCore.processProtectedData({
...
+
+<!-- prettier-ignore-start -->
+```ts twoslash
+import { IExecDataProtectorCore, getWeb3Provider } from '@iexec/dataprotector';
+
+const web3Provider = getWeb3Provider('PRIVATE_KEY');
+const dataProtectorCore = new IExecDataProtectorCore(web3Provider);
+// ---cut---
+
+const taskid = await dataProtectorCore.processProtectedData({
```
Comment (cedric25) [inline tools/dataProtector/dataProtectorCore/processProtectedData.md:227]: Wrong returned variable. (It returns an object)

```
const { taskId } = await dataProtectorCore.processProtectedData({
```

or

```
const processProtectedDataResponse = await dataProtectorCore.processProtectedData({
```

File: tools/dataProtector/dataProtectorCore/protectData.md (hunk 1 above)
Comment (Le-Caignec) [reply tools/dataProtector/dataProtectorCore/protectData.md:163]: i had some warning on my side with the linter as there was not language set

File: tools/dataProtector/dataProtectorCore/grantAccess.md (hunk 2 above)
Comment (Le-Caignec) [reply tools/dataProtector/dataProtectorCore/grantAccess.md:228]: Thanks

File: tools/dataProtector/dataProtectorCore/processProtectedData.md (hunk 3 above)
Comment (Le-Caignec) [reply tools/dataProtector/dataProtectorCore/processProtectedData.md:227]: yeah my bad, a copy paste error

**Pull request merged**